packages/flutter/lib/src/widgets/drag_target.dart

```diff
@@ -182,6 +183,9 @@ class Draggable<T> extends StatefulWidget {
   /// callback is still in the tree.
   final DraggableCanceledCallback onDraggableCanceled;
 
+   /// Called when the draggable is dropped and accepted by a [DragTarget].
+  final VoidCallback onDraggableCompleted;
```

Comment:
these docs should be more detailed, e.g. it should cover information about the lifetime of the widget that provided this callback relative to when the callback is fired, etc.